PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -297,6 +344,22 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
           SETBIT(rejectionMask, EventRejection::Occupancy);
         }
       }
+
+      if (useUpcTrigger) {
+        if (!bcs) {
+          throw std::runtime_error("UPC selection requested but no BC table provided!");
+        }
+        SGCutParHolder sgCuts = setSGPreselection();
+        auto bc = collision.template foundBC_as<BCs>();
+        auto bcRange = udhelpers::compatibleBCs(collision, sgCuts.NDtcoll(), *bcs, sgCuts.minNBCs());
+        auto sgSelectionResult = sgSelector.IsSelected(sgCuts, collision, bcRange, bc);
+        int issgevent = sgSelectionResult.value;
+        if (issgevent > EventTypeUpc::DoubleGap) {
```

Comment:
- Please use easily readable names that follow the naming conventions. The correct name would be `isSgEvent`.
- The name implies a `bool` value which is not the case.
- The variable is used only once which makes it useless and can be replaced with expression itself.